packages/app/cypress/e2e/index.cy.ts

```diff
@@ -129,12 +129,8 @@ describe('App: Index', () => {
           cy.get('@CloseDialogButton').click()
           cy.findByRole('dialog').should('not.exist')
 
-          // TODO Asserts are flakey around the spec page update after scaffolding,
-          // re-evaluate when https://github.com/cypress-io/cypress/pull/19619 is merged
-          // expectedScaffoldPaths.forEach((spec) => {
-          //   // Validate that links for each generated spec are rendered
-          //   cy.get(`a[href="#/specs/runner?file=${spec}"`).should('exist')
-          // })
+          // assert presence of specs list now that specs are found
+          cy.get('.spec-list-container').should('be.visible')
```

Comment:
For the purposes of this test, asserting that the spec-list-container renders at all is enough to prove the workflow has progressed.